robot-db/src/main/java/domain/CategoryName.java

```diff
@@ -0,0 +1,13 @@
+package domain;
+
+/**
+ * Enum with available book categories
+ */
+public enum CategoryName {
+    EDUCATION_AND_SCIENCE,
+    TRAVEL,
+    LIFESTYLE,
+    SEX,
```

Comment:
I truly wonder why that category is here. Truly puzzling. Utterly. Bewildering. No, really.